src/App/screens/Instructor/state/reducers/index.js

```diff
@@ -1,7 +1,7 @@
 import {combineReducers} from 'redux';
 import * as instructorActionTypes from '../actions/instructorActionTypes';
 
-const instructor = (state = false, action) => {
+const instructor = (state = {firstLessonState: 'approved'}, action) => {
```

Comment:
This is just a placeholder right now; not sure if this `firstLessonState` should be computed in redux or on the server? I can do it in redux: fetch all lessons for the instructor, check if there is at least one, grab the first (assuming the new instructor has only uploaded one lesson since they aren't published yet), check it's `state` and use that.